packages/server/src/config.ts

```diff
@@ -26,6 +26,7 @@ export interface MedplumServerConfig {
   approvedSenderEmails?: string;
   database: MedplumDatabaseConfig;
   databaseProxyEndpoint?: string;
+  readonlyDatabase: MedplumDatabaseConfig;
```

Comment:
Seems like we'd want a readonly equivalent of `databaseProxyEndpoint`?

Reply:
Yeah, agreed.  I was curious if RDS Proxy has any of this built in.  That's on my list of things to research.